org.adempiere.base/src/org/compiere/model/MEntityType.java

```diff
@@ -86,16 +86,16 @@ public static MEntityType get(Properties ctx, String entityType)
 	@SuppressWarnings("unused")
 	private static CLogger s_log = CLogger.getCLogger (MEntityType.class);
 	
-    /**
-     * UUID based Constructor
-     * @param ctx  Context
-     * @param AD_EntityType_UU  UUID key
-     * @param trxName Transaction
-     */
-    public MEntityType(Properties ctx, String AD_EntityType_UU, String trxName) {
-        super(ctx, AD_EntityType_UU, trxName);
-    }
-
+    /**
+     * UUID based Constructor
+     * @param ctx  Context
+     * @param AD_EntityType_UU  UUID key
+     * @param trxName Transaction
+     */
+    public MEntityType(Properties ctx, String AD_EntityType_UU, String trxName) {
+        super(ctx, AD_EntityType_UU, trxName);
+    }
+

```

Comment:
saw many of this probably white space only changes, what exactly have been changed here ?

Reply:
I'm not sure why vscode touched those blocks of code. Maybe some auto-linting feature but not clear what it linted? Perhaps these blocks were  ^M terminated ? I've seen that before...but here it's not clear that is the issue.

Reply:
> I'm not sure why vscode touched those blocks of code. Maybe some auto-linting feature but not clear what it linted? Perhaps these blocks were `^M` terminated ? I've seen that before...but here it's not clear that is the issue.

Yes, the same happened to me when editing a file on github, it complained that the file had mixed EOL characters and it was going to be unified.

I checked one of the files and those lines have Linux EOL, while the rest of the file have Windows EOL.